src/components/screens/vacation-requests-screen.tsx

```diff
@@ -417,6 +420,129 @@ const VacationRequestsScreen = () => {
     setLatestVacationRequestStatuses(createdAndFoundVacationRequestStatuses);
   };
 
+  /**
+   * Get a list of working days
+   *
+   * @param loggedInPerson person data
+   */
+  const getWorkingWeek = (loggedInPerson: any): boolean[] => {
+    const workingWeek = [false, false, false, false, false, false, false];
+    const daysOfWeek = ['monday', 'tuesday', 'wednesday', 'thursday', 'friday', 'saturday', 'sunday'];
+  
+    for (let i = 0; i < daysOfWeek.length; i++) {
+      if (loggedInPerson?.[daysOfWeek[i]] !== 0) {
+        workingWeek[i] = true;
+      }
+    }
+    return workingWeek;
+  }
+
+  /**
+ * Calculates vacation days
+ *
+ * @param vacationDayStart vacation start date 
+ * @param vacationDayEnd vacation end date
+ * @param workingWeek list of working days
+ */
+  const calculateTotalVacationDays = (vacationStartDate : DateTime, vacationEndDate : DateTime, workingWeek : boolean []) : number => {
```

Comment:
No space before the colons (:), no space between boolean and []. No need to have an explicit return type as typescript can infer this return. (remove the return type declaration and hover over the function name and see it still has a return type of number.